portal-api/api-tests/api-test-users.js

```diff
@@ -199,22 +202,50 @@ const finder = () => {
   return fluidBuilder;
 };
 
-module.exports.initialise = async (app) => {
+const apiTestUser = {
+  username: 'api-test-user',
+  password: 'P@ssword1234',
+  firstname: 'API',
+  surname: 'Test User',
+  email: 'api-tester@example.com',
+  timezone: 'Europe/London',
+  roles: ['maker'],
+  bank: banks.any,
+};
+
+const setUpApiTestUser = async (as) => {
+  const { salt, hash } = genPassword(apiTestUser.password);
+
+  const userToCreate = {
+    'user-status': 'active',
+    timezone: apiTestUser.timezone || 'Europe/London',
+    salt,
+    hash,
+    ...apiTestUser,
+  };
+
+  userToCreate.password = '';
+  userToCreate.passwordConfirm = '';
+
+  const collection = await db.getCollection('users');
+  await collection.insertOne(userToCreate);
```

Comment:
Do API tests already insert etc.? I've got lost in whether the new endpoint was for mock data loader only or if we are suppose to use it for these tests too. Personally I still prefer this!

Reply:
It looks like there are some other api tests that access the db directly. But they just use it to find a user rather than add one.

I think I agree, since the tests already have direct access to the database, it's best to directly insert the new user rather than going via the `/user` endpoint.